handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -67,6 +68,12 @@
     static final Set<String> SUPPORTED_PROTOCOLS_SET;
     static final String[] EXTRA_SUPPORTED_TLS_1_3_CIPHERS;
     static final String EXTRA_SUPPORTED_TLS_1_3_CIPHERS_STRING;
+    static final String[] NAMED_GROUPS;
+
+    // Use default that is supported in java 11 and earlier and also in OpenSSL / BoringSSL.
+    // See https://github.com/netty/netty-tcnative/issues/567
+    // See https://www.java.com/en/configure_crypto.html for ordering
+    private static final String[] DEFAULT_NAMED_GROUPS = { "x25519", "secp256r1", "secp384r1", "secp521r1" };
```

Comment:
JDK11 also supports X448: https://www.oracle.com/java/technologies/javase/11-0-9-relnotes.html
Consider adding it to the default list.

Reply:
its not supported by BoringSSL